modules/settings/assets/js/hooks/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as useAuth } from './use-auth';
+export { default as useModal } from './use-modal';
```

Comment:
'default' is not needed, just use the name for export like { useAuth }

Reply:
updated